plutus-core/plutus-core/test/CostModelInterface/Spec.hs

```diff
@@ -0,0 +1,158 @@
+{-# LANGUAGE FlexibleInstances    #-}
+{-# LANGUAGE LambdaCase           #-}
+{-# LANGUAGE OverloadedStrings    #-}
+{-# LANGUAGE TypeApplications     #-}
+{-# LANGUAGE TypeSynonymInstances #-}
+
+module CostModelInterface.Spec (test_costModelInterface) where
+
+import           PlutusCore
+import           PlutusCore.Evaluation.Machine.CostModelInterface
+import           PlutusCore.Evaluation.Machine.ExBudget
+import           PlutusCore.Evaluation.Machine.MachineParameters
+
+import qualified Data.Map                                         as Map
+import qualified Data.Text                                        as Text
+import           Test.Tasty
+import           Test.Tasty.HUnit
+
+type CekCostModel = CostModel CekMachineCosts
+
+-- Just for testing
+randomCekCosts :: CekMachineCosts
+randomCekCosts =
+    CekMachineCosts { cekStartupCost = ExBudget 2342 234321
+                    , cekVarCost     = ExBudget 12312 56545
+                    , cekConstCost   = ExBudget 23490290838 2323423
+                    , cekLamCost     = ExBudget 0 712127381
+                    , cekDelayCost   = ExBudget 999 7777
+                    , cekForceCost   = ExBudget 1028234 0
+                    , cekApplyCost   = ExBudget 324628348 8273
+                    , cekBuiltinCost = ExBudget 4 4
+                    }
+
+cekVarCostCpuKey :: Text.Text
+cekVarCostCpuKey = "cek_var_cost-_ex_budget_cpu"  -- flatten . camelToSnake
```

Comment:
Commented out code.

Reply:
> Commented out code.

That's supposed to be a comment to explain where the weird string's come from.